### DISCUSSION TITLE
Recover Deleted Codespace

### DISCUSSION THREAD

**Discussion by ghost** [community/community#44365] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Last night while stopping my codespace, I bymistakely deleted that. By searching I didn't get any option to restore deleted ones. The codespace is made from blank codespace there was no repo, after finishing my work I decided to push to a new repo. Please recover my last deleted codespace. This was the codespace id `swayam25-glorious-adventure-7674xx6qq6vhr745-2500`.

**Comment by salvador-barboza** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). In the future, please open a support ticket. For now, we restored you Codespace.

**Reply by ghost** [reply]
Ok
